com.github.AberothCommunity.Aberoth.yaml

```diff
@@ -0,0 +1,44 @@
+app-id: com.github.AberothCommunity.Aberoth
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk8
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+  - --persist=Downloads
+  - --env=PATH=/app/bin:/app/jre/bin:/usr/bin
+  - --env=JAVA_HOME=/app/jre
+command: aberoth
+
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk8/install.sh
+
+  - name: aberoth
+    buildsystem: simple
+    build-commands:
+      - install -D com.github.AberothCommunity.Aberoth.sh /app/bin/aberoth
+      - install -Dm644 -t /app/share/applications com.github.AberothCommunity.Aberoth.desktop
+      - install -Dm644 -t /app/share/icons/hicolor/scalable/apps com.github.AberothCommunity.Aberoth.svg
+      - install -Dm644 -t /app/share/metainfo com.github.AberothCommunity.Aberoth.metainfo.xml
+    sources:
+      - type: file
+        path: com.github.AberothCommunity.Aberoth.desktop
+      - type: file
+        path: com.github.AberothCommunity.Aberoth.sh
+      - type: file
+        path: com.github.AberothCommunity.Aberoth.svg
+      - type: file
+        path: com.github.AberothCommunity.Aberoth.metainfo.xml
+      - type: extra-data
+        filename: Aberoth.jar
+        sha256: 71d204ac348b5a133ee086894be74313d2ee0eb08585c58c93163ed18e09c9f8
+        size: 57113
+        url: https://aberoth.com/resource/Aberoth.jar
```

Comment:
A versioned download link is not available?

Reply:
No the game updates automatically.

Reply:
this is not gonna go well. the sha256 will change and the package won't be able to be installed.

Reply:
Autoupdates won't work in flatpak. The problem is aberoth.jar may silently change on the server and the shasum won't match making builds fail. You have to change it each time.

Reply:
Aberoth.jar is the updater it's never changed.

Reply:
> Aberoth.jar is the updater it's never changed.

So where's the actual game?

Reply:
it's "~/Downloads/AberothClient.jar"